### DISCUSSION TITLE
ch8/ch8-03

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#217] [General]
# ch8/ch8-03

<The Go Programming Language>中文版

https://gopl-zh.github.io/ch8/ch8-03.html

**Comment by 8vd8** [comment]
不是只有我感觉这本书写的很一般嘛

**Reply by ShiinaTaki2027** [reply]
有点太跳跃了，不是给纯新手写的qaq

**Reply by a-firefly** [reply]
我觉得写的挺好，层层递进，讲的知识点也挺全面。

**Comment by identifysun** [comment]
经过测试发现，`netcat2` 程序在 M3 电脑上无法正常工作，相关地址如下：

https://stackoverflow.com/questions/78996496/io-copyconn-os-stdin-do-not-work-as-expected

**Reply by zishiguo** [reply]
M1 Pro页无法正常工作。链接404了

**Reply by a-firefly** [reply]
我也遇到了，我是 M1 上无法正常工作，不等输入内容，程序就直接退出了。我的 go 版本是 1.23.2
我把 go 升级到 1.24.2 之后，这个问题就解决了。
